infer/models/c/src/libc_basic.c

```diff
@@ -682,6 +682,9 @@ unsigned sleep(unsigned seconds) {
 #ifdef __APPLE__ // define __WAIT_STATUS as int * on mac
 #define __WAIT_STATUS int*
 #endif
+#ifdef __GNUC__  // define __WAIT_STATUS as int * with glibc
```

Comment:
Actually I need to investigate it more. It looks like __WAIT_STATUS is a struct on some systems (headers that come with old gcc for example), please hold off with changes until I find out more